.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -0,0 +1,89 @@
+name: Bug Report
+description: Found something you weren't expecting?  Report it here!
+body:
+- type: markdown
+  attributes:
+    value: |
+      NOTE: If your issue is a security concern, please send an email to security@gitea.io instead of opening a public issue.
+- type: markdown
+  attributes:
+    value: |
+      1. Please speak English, this is the language all maintainers can speak and write.
+      2. Please ask questions or configuration/deploy problems on our Discord
```

Comment:
Should this still be here now that we have it already on the new issue landing page with the config.yaml?

Reply:
Always useful to have this information in multiple locations.  It also keeps it on the page where they are submitting the form.